src/braket/aws/aws_session.py

```diff
@@ -176,8 +182,7 @@ def _notebook_instance_version() -> str:
         )
 
     def add_braket_user_agent(self, user_agent: str) -> None:
-        """
-        Appends the `user-agent` value to the User-Agent header, if it does not yet exist in the
+        """Appends the `user-agent` value to the User-Agent header, if it does not yet exist in the
```

Comment:
nit: inconsistent capitalization for user-agent

Reply:
In this case, one is referencing a value passed in and the other references the noun `User-Agent`

Reply:
Hm, then should the method above be "User-Agent header" without the backticks as well? Or backticks in both places

Reply:
I think that is a fair callout - changing this to use `User-Agent` in both docstrings